Source/GSCodingFlags.h

```diff
@@ -0,0 +1,385 @@
+/*
+   GSCodingFlags.h
+
+   Define flags used in keyed coding. Extracted from existing files.
+
+   Copyright (C) 2019 Free Software Foundation, Inc.
+
+   Author: Fred Kiefer <fredkiefer@gmx.de>
+   Date: 12.2019
+
+   This file is part of the GNUstep GUI Library.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; see the file COPYING.LIB.
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110-1301, USA.
+*/
+#ifndef _GNUstep_H_GSCodingFlags
+#define _GNUstep_H_GSCodingFlags
+
+#import "GNUstepGUI/GSNibLoading.h"
+
+typedef struct _GSCellFlags {
+#if GS_WORDS_BIGENDIAN == 1
```

Comment:
OK, I will do so.

Reply:
I'm probably missing something, but I don't see this is documented in the `.h` file yet?

This:

> Define flags used in Cocoa for keyed coding.

doesn't to me, as a casual reader, suggest "we are really trying to binary compatible with flags used in Cocoa for some values during keyed coding"? :/

Reply:
It does for me :-)
Give me some text to put there and I will do so.

Reply:
Perhaps something like:

"The following code is architecture specific.  On different platforms (big endian vs little endian) the byte ordering is different.   This is done here so that bitwise operations to determine the state of certain flags works correctly."

Not sure if that is concise enough, but it does describe why it's done.

Reply:
This would explain what the code does but not why it does that. And the code itself already states that. To me this would violate one of the fundamental rules on code comments: Not to duplicate the code.